src/components/form/FieldBase.vue

```diff
@@ -108,6 +110,12 @@ export default {
     unlabelled() {
       return this.label === '';
     },
+    labelId() {
```

Comment:
This is the change to tackle the `id` issue I mentioned in the other PR, right?

Ohhh, I just saw the `aria-labelledby` in the component. Now I got it hehehe
As it is something that caused some misunderstandings already, I would recommend to cover it with a test (HdCheckbox case).

Reply:
Yep, that's it.
Unit test added 👍